### PR TITLE
Use new and better canonical link to Google Groups

### PR DESCRIPTION
**Pull request by xuanswe**
https://github.com/etcd-io/website/issues/479

### CODE REVIEW

**Review by hexfusion — APPROVED** [review]

**Comment by xuan-nguyen-swe** [comment]
Hi @hexfusion is there anything that prevents merging this PR?

**Comment by hexfusion** [comment]
I don't have merge perms or I would sorry.

@serathius @spzala

**Review by jmhbnz — APPROVED** [review]

**Review by spzala — CHANGES_REQUESTED** [review]
@xuan-nguyen-swe thank you for catching this, lgtm but I have one comment - I don't think any of the etcd project use IRC node now and we should use Slack channel as used by other etcd repos. I would suggest to change it "* Slack: [#etcd](https://kubernetes.slack.com/messages/C3HD8ARJ5/details/) channel on Kubernetes ([get an invite](http://slack.kubernetes.io/))" something that we are using in etcd main repo. Does that make sense? Thanks!

**Comment by spzala** [comment]
> I don't have merge perms or I would sorry.
> 
> @serathius @spzala

Thanks for the review and ping @hexfusion !! I just reviewed and we will make sure it gets merged.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ contribution. See the [DCO](DCO) file for details.
 # Email and Chat
 
 The project currently uses the general CoreOS email list and IRC channel:
```
Comment (spzala) [inline CONTRIBUTING.md:17]: Also, this sentence with `the general CoreOS email list and IRC channel` needs to be modified according to the proposed changes. Thanks!

**Review by spzala — CHANGES_REQUESTED** [review]

**Review by gyuho — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (xuan-nguyen-swe) [reply CONTRIBUTING.md]: Hi @spzala,

Thanks for your comment. It should be in synced with the info in main repo now.

**Review by spzala — APPROVED** [review]
Thanks for addressing my comments @xuan-nguyen-swe LGTM.

**Pull request merged**